overrides/cf.d.ts

```diff
@@ -1,6 +1,14 @@
 // TODO: add Response cf object too
 
 declare class Request extends Body {
+  /**
+   * In addition to the properties on the standard `Request` object,
+   * the `cf` object contains extra information about the request provided
+   * by Cloudflare's edge.
+   *
+   * Note: Currently, settings in the `cf` object cannot be accessed in the
+   * playground.
+   */
   readonly cf?: IncomingRequestCfProperties;
```

Comment:
This should be a getter (i.e. `get cf`)

Reply:
I think so, but for consistency with the rest of the types, which are all getters